db_management/db_upgrade.rst

```diff
@@ -0,0 +1,418 @@
+
+Get started
+===========
+
+The upgrade process varies depending on where your database is hosted.
+
+.. _db-upgrade/online:
+
+Online (SaaS)
+-------------
+
+If you are hosted Online, please check our :ref:`instruction page for our Online (SaaS) customers
+<upgrade_button>`.
+
+.. _db-upgrade/odoo-sh:
+
+Odoo.sh
+-------
+
+If you are Odoo.sh hosted, check our :doc:`specific instructions to be able to upgrade
+<../odoo_sh/advanced/upgrade_your_database>`.
+
+.. _db-upgrade/on-premise:
+
+On-Premise
+----------
+
+There are two possibilities:
+
+#. Via the interface of our `website form <https://upgrade.odoo.com>`_
+#. | For technically-advanced users and partners, via the following command line (to be used on the
+     machine where your database is hosted):
+   | ``python <(curl -s beta.upgrade.odoo.com/upgrade) test -d <your db name> -t 14.0 -c <your
+     subscription code>``
+
+What does it do?
+~~~~~~~~~~~~~~~~
+
+The above command will dump your database to a file, then send it to the upgrade platform for
+upgrade, displaying you the live logs, then restore the upgraded database back on your server as a
+duplicate test database.
+
+.. _db-upgrade/steps:
+
+Steps
+=====
+
+.. _db-upgrade/steps-test:
+
+The testing phase
+-----------------
+
+.. _db-upgrade/test-process:
+
+Test process
+~~~~~~~~~~~~
+
+Also referred to as the pre-production phase, the test phase allows you to review an upgraded
+version of your database without affecting your production database in any way.
+
+We suggest that you run the test upgrade process at least once, but you can do it as often as you
+want (one at a time).
+
+Once you receive your upgraded test database, you should check that all data, processes and
+functionality are still correct and working as expected.
+
+If you do find discrepancies, you'll be able to:
+
+* | :ref:`Report your issues <db-upgrade/test-assistance>`
+  | and/or
+* Ask for a new :ref:`test request <db-upgrade/test-db-request>` after the reported issues have
+  been fixed in the upgrade script.
+
+When you do not find any discrepancies, you'll be able to:
+
+* Move on to the upgrade of your :ref:`production database <db-upgrade/production-live>`.
+
+.. _db-upgrade/test-db-request:
+
+Request a test database
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When filling the `website form <https://upgrade.odoo.com>`_, select *Testing* purpose.
+
+.. image:: media/db-upgrade-test-purpose.png
+   :align: center
+   :alt: Selection of the "Testing" purpose in the upgrade form on Odoo
+
+.. _db-upgrade/test-guidance:
+
+Test guidance
+~~~~~~~~~~~~~
+
+Every business and organization has its own operational needs and will have to test its specific
+Odoo instance respectively. However, we recommend you look at `the test scenario
+<https://drive.google.com/open?id=1Lm4JqbsHBirB1wMi14UChoz_YHLjx5ec>`_ we created, a high-level idea
+of what you should test and look out for.
+
+.. todo:: change link "test scenario" once the related doc is published
```

Comment:
Shouldn't it be done before merging?

Reply:
@KangOl 
it's another doc that isn't in the repo yet.
So, until then, the google link is the valid one.